MobileFilter/sections/specific_web.txt

```diff
@@ -1030,13 +1023,11 @@ opencritic.com,www-news18-com.cdn.ampproject.org,eurogamer.net,mumsnet.com,news1
 jianshu.com##.dt-open-bg[data-scene="leftBtn"]
 jianshu.com##.note-comment-above-ad-wrap
 theinventory.com,thetakeout.com,theonion.com,splinternews.com,clickhole.com,avclub.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,theroot.com##main > div > script[type="application/ld+json"] ~ div[class*="-1 "][class*="-0 "]
-theinventory.com,thetakeout.com,theonion.com,splinternews.com,clickhole.com,avclub.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,theroot.com##.inline-ad-container
```

Comment:
![image](https://user-images.githubusercontent.com/8361299/116092989-f8958180-a6ae-11eb-8ffb-fd43c419aed3.png)
it is useful

Reply:
If you about two rules - they are grouped. Better leave as is.

Reply:
It seems it is required on theinventory.com only

Reply:
Okay.